### PR TITLE
Fixes for #51

### PR DESCRIPTION
**Pull request by lkhrs**
- [x] redirects (lkhrs said this can be added by 11ty)
- [x]  feed.xml
- [x]  /credits link to patreon
- [x]  sidebar is underlined (happened when added underlined to li a links)
- [x]  /docs is missing its second indent bullets (missing the css for it)

#51

### CODE REVIEW

**Comment by anjakefala** [comment]
I can confirm that the underlined links on /credits is working: https://deploy-preview-52--distracted-goldwasser-45d076.netlify.app/credits/ along with the non-underlining of the sidebar on /docs: https://deploy-preview-52--distracted-goldwasser-45d076.netlify.app/docs/

**Comment by anjakefala** [comment]
Also, the nested bullets now work! Thank you so much. =)

File: .eleventy.js [hunk 1]
```diff
@@ -128,6 +128,9 @@ module.exports = function(eleventyConfig) {
     eleventyConfig.addPassthroughCopy({ "./node_modules/asciinema-player/resources/public/js/asciinema-player.js": "./asciinema-player.js" });
     eleventyConfig.addPassthroughCopy({ "./node_modules/asciinema-player/resources/public/css/asciinema-player.css": "./asciinema-player.css" });
 
+    // Pass through the redirects
+    eleventyConfig.addPassthroughCopy({ "./redirects.tsv": "./_redirects" });
```
Comment (lkhrs) [inline .eleventy.js:132]: Added config line to tell 11ty to copy `redirects.tsv` to `_site/_redirects`

11ty Passthrough File Copy: https://www.11ty.dev/docs/copy/

File: site/includes/layouts/page.njk [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 ---
-<div id="container" class="py-10 w-full">
+<article id="container" class="py-10 w-full">
```
Comment (lkhrs) [inline site/includes/layouts/page.njk:4]: Our CSS selector that styles the `a` links (line 13 in `styles/tailwind.css` assumes the links are inside an `article` element, which was missing from the `includes/layouts/page.njk` layout, or `page` layout.

Layouts are specified in the front matter of any .md, .njk, or .html page in site/. We currently have the following layouts:
`docs`, which gives the page that table of contents sidebar on the docs pages and applied automatically to any .html placed in site/docs/*
`page`, used for any generic page
`post`, used for any post and applied automatically to any .md file placed in site/blog/*
`default`, the global layout that `page` extends - I don't recommend using this layout by itself as `page` adds the title from the front matter and the `<article>` element around the content.

File: styles/tailwind.css [hunk 3]
```diff
@@ -10,7 +10,7 @@ p a, article a {
   @apply border-brand-primary border-b-2;
 }
 
-ul a, article a {
+article a {
```
Comment (lkhrs) [inline styles/tailwind.css:13]: removed `ul a` as it is too wide of a selector and was affecting the `docs` layout

File: styles/tailwind.css [hunk 4]
```diff
@@ -41,6 +41,13 @@ p {
 article ul {
   @apply list-disc;
 }
+li ul {
+  @apply list-inside;
+  list-style-type: circle;
+}
+aside ul {
+  list-style-type: none;
+}
```
Comment (lkhrs) [inline styles/tailwind.css:50]: Indented lists use the following HTML format:
```
<ul>
<li></li>
<li>
   <ul>
     <li></li>
   <ul>
</li>
</ul>
```

So to style them, we select `li ul` and then apply the styles. Here I used the `list-inside` Tailwind utility classes to indent the child list and then the CSS property `list-style-type` to give the bullets that hollow circle look.

However, this affects the `docs` sidebar, which is wrapped in an `aside` element, so I set any `ul` within an `aside` element to have no `list-style-type`, which hides the bullets.

**Pull request merged**